### ISSUE TITLE
Trying to upload images crashes the app

### ISSUE THREAD

**Issue by joashrajin** [wordpress-mobile/WordPress-Android#19638]
### Expected behavior

The app should not be crashing and should allow you to upload/select images

### Actual behavior

The app keeps crashing when you try to access the Media Library or select an image from your device. 

### Steps to reproduce the behavior

https://github.com/wordpress-mobile/WordPress-Android/assets/16253818/12e99fff-2605-43fb-b1c9-34533c2e00ab

1. Create a new blog post
2. Attempt to add an image block
3. Try to select an image from your device or media library, it does not work

**Comment by joashrajin** [comment]
7277111-zd-a8c - crash is in sentry. Link in the ticket

**Comment by joashrajin** [comment]
[Sentry](https://a8c.sentry.io/discover/jetpack-android:c08b9c02582b4870bb79f8dc65b2755a/?field=title&field=event.type&field=project&field=user.display&field=timestamp&field=replayId&homepage=true&name=All+Events&project=5731682&query=user.email%3Amail%40theolounge.de&sort=-timestamp&statsPeriod=30d&yAxis=count%28%29)

**Comment by irfano** [comment]
Duplicate of https://github.com/wordpress-mobile/WordPress-Android/issues/19157

**Comment by derekblank** [comment]
Related issue and discussion:
* https://github.com/wordpress-mobile/WordPress-Android/issues/9685

**Comment by irfano** [comment]
@joashrajin, can you consistently reproduce this, even after a clean install?
The crash logs show a memory-related crash, but I suspect it can be related to permissions, as it crashed before receiving the permission. I can't see a reason that the memory can be the issue without having permission to fetch medias.

**Comment by joashrajin** [comment]
> @joashrajin, can you consistently reproduce this, even after a clean install? The crash logs show a memory-related crash, but I suspect it can be related to permissions, as it crashed before receiving the permission. I can't see a reason that the memory can be the issue without having permission to fetch medias.

Yep, I uninstalled and tried a clean install of v23.3.1 and it works fine. Clean installed 23.4 and the issue started. Updated to the latest beta and the issue persists. 

I have tried giving all permissions to the app but that made no difference. 

Something interesting is that the crash only occurs on my WP.com sites, not on my Pressable test sites.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
